website/content/docs/discovery/dns.mdx

```diff
@@ -285,6 +285,50 @@ caching of DNS results. However, there are many situations in which caching is
 desirable for performance and scalability. This is discussed more in the tutorial
 for [DNS caching](https://learn.hashicorp.com/tutorials/consul/dns-caching).
 
+## DNS with ACLs
+
+After enabling [Access Control Lists (ACLs)](/docs/security/acl/acl-system),
+DNS requests will fail with error `NXDOMAIN` unless ACL tokens have been
+configured with the necessary rules.
+
+When a DNS request is made to a Consul agent, the agent will use a previously
+configured token with the request (since there is no means for the initial DNS
+request to specify a token):
+
+- The agent's [`default` token](/docs/agent/options#acl_tokens_default)
+  token, if configured.
+- Otherwise, the built-in [`anonymous` token](/docs/security/acl/acl-system#builtin-tokens).
+  Because the anonymous token is used when any request is made to Consul without
+  explicitly specifying a token, production deployments should not apply policies
+  needed for DNS to this token.
+
+Consul will either authorize or revoke the request, depending on the token's
+policy rules:
```

Comment:
```suggestion
Consul will either authorize the request, depending on the token's
policy rules:
```

Reply:
Why "authorize" instead of "either authorize or deny" (as Adam suggested)?

Reply:
After reading this a second time, it should probably say that Consul will accept or reject the request depending on whether the token has appropriate authorization.

Authorization technically takes place when the operator creates the policy and applies it to the token. Policy enforcement, which is what this is referring to, is where access is granted or rejected based on permissions granted by the policy.